src/containers/Auth/ConfirmOTP/ConfirmOTP.module.css

```diff
@@ -101,3 +101,39 @@
   font-weight: bold;
   font-size: 14px;
 }
+
+.Form {
```

Comment:
We are adding this in every css file. Let's not do this. We should have common css in Auth css